lib/actions/react.js

```diff
@@ -0,0 +1,8 @@
+module.exports = function (github, payload, react) {
+  return github.reactions.createForIssue({
+    user: payload.repository.owner.login,
+    repo: payload.repository.name,
+    number: payload.issue.number,
+    content: react
+  });
```

Comment:
Instead of passing the `user`, `repo`, and `number` params, you can now just call `payload.toIssue({content: react})` and it will return an object with everything merged together. See the [label](https://github.com/bkeepers/PRobot/blob/master/lib/actions/label.js) action for an example.